cmd/zrepl/mgmt_conn.c

```diff
@@ -821,13 +821,25 @@ process_message(uzfs_mgmt_conn_t *conn)
 		strncpy(zvol_name, payload, payload_size);
 		zvol_name[payload_size] = '\0';
 
-		if (((zinfo = uzfs_zinfo_lookup(zvol_name)) == NULL) ||
-		    (zinfo->mgmt_conn != conn)) {
```

Comment:
this condition makes sure that commands are received on correct connection, and no issue happened in replicas communicating with target. Removing this condition can lead to any issues due to any issues with replicas or targets.
Its better to have this condition.

Reply:
oh sorry.. got to see this in next line.. :) :+1:

Reply:
and also the refcount issue.. thanks for fixing this @jkryl